config/locales/en_BD.yml

```diff
@@ -1,4 +1,5 @@
 en_BD:
+  facility_group: "upazila"
```

Comment:
@claudiovallejo Facility groups are equivalent to districts in BD, not upazillas, correct?

Reply:
@rsanheim You shared this matrix on Slack, which looks correct to me.

```
| codebase Name    | India        | Bangladesh   | Ethiopia     | Generic name
|—————————+———————+———————+———————|
| Organization     | Organization | Organization | Organization | Level 1
| State            | State        | District     | Region       | Level 2
| District         | District     | Upazila      | Zone         | Level 3
| Block (aka Zone) | Block        | Union        | Woreda       | Level 4
| Facility         | Facility     | Facility     | Facility     | Level 5
```

Reply:
@rsanheim @dburka Facility groups are equivalent to Upazilas in Bangladesh. When adding a new facility to an Upazila, the user should be able to select a Union.

Reply:
Thanks for the clarification, I should've just referred to that chart =).